api/src/indicators.py

```diff
@@ -137,38 +137,45 @@ def patch_indicator(
     )
 
 
+def add_matched_queries_keyword(hit):
+    hit["matched_queries"] = ["keyword_search"]
+    return hit
+
+def add_matched_queries_semantic(hit):
+    hit["matched_queries"] = ["semantic_search"]
+    return hit
+
+
 @router.get(
     "/features/search", response_model=IndicatorSchema.FeaturesSemanticSearchSchema
 )
 def semantic_search_features(
-    query: Optional[str], size=10, scroll_id: Optional[str] = None
+    query: str, size=10 # , scroll_id: Optional[str] = None
```

Comment:
Can't receive or return a scroll id- which scroll_id should we use if we are merging the results of two queries manually? I don't think anyone was scrolling with the semantic search endpoint (dojo wasn't), but we can confirm with Uncharted.